src/index/ExternalVocabulary.cpp

```diff
@@ -56,6 +59,27 @@ void ExternalVocabulary::buildFromVector(const vector<string>& v,
   initFromFile(fileName);
 }
 
+// _____________________________________________________________________________
+void ExternalVocabulary::buildFromTextFile(const string& textFileName,
+                                         const string& outFileName) {
+  _file.open(outFileName.c_str(), "w");
```

Comment:
This is missing a check if the `open()` actually worked. I just ran this in docker and had wrong permissions on the volume where the index is created and only when creating the real index did I see it fail.

Reply:
Fixed